Cargo.toml

```diff
@@ -26,6 +26,8 @@ cfg-if = "1.0.0"
 chrono = "0.4.19"
 clap = { version = "4", features = ["derive"] }
 config = "0.13.3"
+#ear = "0.2.0"
```

Comment:
This is because the latest commits to rust-ear broke the crate (Appraisal isn't `send` anymore`). Before we merge this, the crate will need to be fixed and i will fix the version.